qiskit_ibm_provider/job/ibm_circuit_job.py

```diff
@@ -595,24 +607,24 @@ def refresh(self) -> None:
             IBMJobApiError: If an unexpected error occurred when communicating
                 with the server.
         """
+        # TODO: Change to use runtime response data as much as possible
         with api_to_job_error():
-            api_response = self._api_client.job_get(self.job_id())
+            api_response = self._runtime_client.job_get(self.job_id())
 
         try:
-            api_response.pop("job_id")
-            self._creation_date = dateutil.parser.isoparse(
-                api_response.pop("creation_date")
-            )
-            self._api_status = api_response.pop("status")
-            if "kind" in api_response:
+            api_response.pop("id")
+            self._creation_date = dateutil.parser.isoparse(api_response.pop("created"))
+            self._api_status = api_response.pop("state")["status"]
+            if "kind" in api_response:  # not relevant in runtime?
                 self._kind = ApiJobKind(api_response.pop("kind"))
-            if "qobj" in api_response:
+            if "qobj" in api_response:  # not relevant in runtime?
                 self._qobj = dict_to_qobj(api_response.pop("qobj"))
         except (KeyError, TypeError) as err:
             raise IBMJobApiError(
                 "Unexpected return value received " "from the server: {}".format(err)
             ) from err
 
+        # not relevant in runtime?
```

Comment:
A lot of these aren't returned by runtime API. Please check the API doc, or better yet, test it yourself as I'm not convinced the doc is 100% up to date.